File: OldStory/tmp/ecdf.m
%subplot(2, 1, 1)

o = csvread("ecdf_original.csv")(:, [2, 3]);
d = csvread("ecdf_dropped.csv")(:, [2, 3]);
plot(o(:, 1), o(:, 2), 'rx', 'markersize', 2, d(:, 1), d(:, 2), 'bo', 'markersize', 2)
legend("Original", "Modified")

%subplot(2, 2, 3)
%o = csvread("original.csv");
%d = csvread("dropped.csv");
%[on ov] = hist(o, 100);
%[dn dv] = hist(d, 100);
%plot(ov, on, 'r', dv, dn, 'b')

%subplot(2, 2, 4)
%[on ov] = hist(o, 50);
%[dn dv] = hist(d, 50);
%plot(ov, on, 'r', dv, dn, 'b')

File: OldStory/tmp/ecdf.R
o <- read.csv("original.csv", header = FALSE, stringsAsFactor = FALSE)
d <- read.csv("dropped.csv", header = FALSE, stringsAsFactor = FALSE)
o <- o$V1
d <- d$V1

fo <- ecdf(o)
fd <- ecdf(d)

mato <- cbind(unique(o), fo(unique(o)))
matd <- cbind(unique(d), fd(unique(d)))
write.csv(mato, file = "ecdf_original.csv")
write.csv(matd, file = "ecdf_dropped.csv")
